Json.More/JsonElementExtensions.cs

```diff
@@ -134,6 +135,8 @@ void ComputeHashCode(JsonElement obj, ref int current, int depth)
 	/// <remarks>
 	/// See https://github.com/dotnet/runtime/issues/42502
 	/// </remarks>
+	[RequiresUnreferencedCode("Calls System.Text.Json.JsonSerializer.Serialize<T>(T)")]
```

Comment:
Pretty sure the annotations could be avoided here -- these methods are dealing with known types (calls to `JsonSerializer` can be avoided altogether).

Reply:
Not sure how serializer calls could be avoided.  The point of this method is that `.ToString()` doesn't give you valid JSON, and `.GetRawText()` only gives the text that was input, including all of the whitespace; you can't perform other formatting.

If the serializer _can_ be avoided, I'm interested in that.  I mean, I could manually build it, but I prefer not to.

The other option here is to create a serializer context and toss `JsonElement` onto it, then just pass that in here.